koku/masu/external/downloader/gcp/gcp_report_downloader.py

```diff
@@ -206,9 +207,14 @@ def _get_relevant_file_names(self, start_date, end_date):
         dates_as_strings = [date.strftime('%Y-%m-%d') for date in dates]
         available_file_names = self._get_bucket_file_names()
         relevant_file_names = set()
+        prefix = ''
```

Comment:
Collapse these lines into a ternary?
```suggestion
        prefix = f'{self.report_prefix}-' if self.report_prefix else ''
```
Just a style suggestion.